food-truck-api/src/main/java/food/truck/api/truck/TruckRepository.java

```diff
@@ -0,0 +1,12 @@
+package food.truck.api.truck;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface TruckRepository extends JpaRepository<Truck, Long> {
```

Comment:
I'm assuming these methods' bodies are implemented by Spring? If so, wow... they really did go overboard with making Spring code write itself.

Reply:
As far as I understand the default behavior is for it to create implementations for all subinterfaces, so yes.